components/MetadataEditor.vue

```diff
@@ -0,0 +1,407 @@
+      return await this.$api.library.getAncestors({
+        itemId: this.metadata.Id as string,
+        userId: this.$auth.user.Id
+      });
+    },
+    async saveMetadata() {
+      const item = pick(this.metadata, [
+        'Id',
+        'Name',
+        'OriginalTitle',
+        'ForcedSortName',
+        'CommunityRating',
+        'CriticRating',
+        'IndexNumber',
+        'AirsBeforeSeasonNumber',
+        'AirsAfterSeasonNumber',
+        'AirsBeforeEpisodeNumber',
+        'ParentIndexNumber',
+        'DisplayOrder',
+        'Album',
+        'AlbumArtists',
+        'ArtistItems',
+        'Overview',
+        'Status',
+        'AirDays',
+        'AirTime',
+        'Genres',
+        'Tags',
+        'Studios',
+        'PremiereDate',
+        'DateCreated',
+        'EndDate',
+        'ProductionYear',
+        'AspectRatio',
+        'Video3DFormat',
+        'OfficialRating',
+        'CustomRating',
+        'People',
+        'LockData',
+        'LockedFields',
+        'ProviderIds',
+        'PreferredMetadataLanguage',
+        'PreferredMetadataCountryCode',
+        'Taglines'
+      ]);
+      try {
+        this.loading = true;
+        await this.$api.itemUpdate.updateItem({
+          itemId: this.metadata.Id as string,
+          baseItemDto: item
+        });
+        this.$emit('save');
+        this.loading = false;
+        this.$store.dispatch('snackbar/display', {
+          message: 'saved',
+          color: 'success'
+        });
+      } catch (error) {
+        let errorMessage = this.$t('unexpectedError');
+
+        if (error.response.status === 400) {
+          errorMessage = this.$t('badRequest');
+        }
+
+        this.$store.dispatch('snackbar/display', {
+          message: errorMessage.toString(),
+          color: 'error'
+        });
+      }
+    },
+    saveDate(key: string, date: string) {
+      this.menu = false;
+      set(this.metadata, key, this.$dateFns.formatISO(new Date(date)));
+    },
+    handlePersonEdit(item: BaseItemPerson | null = null) {
+      this.person = item;
+      this.dialog = true;
+    },
+    handlePersonUpdate(item: BaseItemPerson) {
+      if (!this.metadata.People) {
+        this.metadata.People = [];
+      }
+      const target = this.metadata.People?.find(
+        (person) => person.Id === item.Id
+      );
+      if (target) {
+        Object.assign(target, item);
+      } else {
+        this.metadata.People.push(item);
+      }
+    },
+    handlePersonDel(index: number) {
+      (this.metadata.People as BaseItemPerson[]).splice(index, 1);
+    }
+  }
+});
+</script>
+<style scoped>
+.metadata-editor {
+  position: relative;
+}
+.person-icon {
+  background-color: var(--v-secondary-darken1);
+}
+.headline {
+  border-bottom: 1px solid var(--v-secondary-lighten1);
+}
+.metadata-editor >>> .v-tab {
+  border-right: 1px solid var(--v-secondary-lighten1);
+}
+.metadata-editor >>> .v-card__actions {
+  position: absolute;
+  bottom: 10px;
```

Comment:
```suggestion
```
This is causing issues for the action icons in the image editor to overlay onto the text